models/report.js

```diff
@@ -0,0 +1,19 @@
+module.exports = (sequelize, DataTypes) => {
+  const Report = sequelize.define('Report', {
+    report: {
+      type: DataTypes.TEXT,
+      allowNull: false,
+    },
+    userId: {
+      type: DataTypes.STRING,
```

Comment:
This is where the problem is. The `userId` type should be `INTEGER`, not `STRING`

Reply:
wow, thanks.

Reply:
I've pushed the correction